### ISSUE TITLE
Docs note on STS + IAM

### ISSUE THREAD

**Issue by jantman** [jantman/awslimitchecker#139]
We should note in the docs that when using temporary credentials (i.e. federated login credentials, retrieved from IAM's GetFederationToken action, support for which is requested in #123 ) with a SessionToken, the IAM API does not work. It will fail with an invalid token error.

I don't have a docs pointer for this, though I've asked for one. All I have is the response from AWS support:

```
Q: (1) Do STS tokens simply not work with the IAM API, even if they have permissions for it? I've searched the documentation extensively, but can't find any mention of this.

A: It’s because the access key id you are using (the temp ones created) do not match the ones in our records for that user. That’s by design. I've found another similar case: https://github.com/99designs/aws-vault/issues/15
```

**Comment by jantman** [comment]
AWS Support said they've reached out to the docs team to get this documented somewhere. I guess I'll take some periodic looks at the STS and IAM docs to see if that information shows up.

**Comment by jantman** [comment]
http://docs.aws.amazon.com/STS/latest/APIReference/API_GetFederationToken.html

>The temporary security credentials created by GetFederationToken can be used to make API calls to any AWS service with the following exceptions:
>
>    You cannot use these credentials to call any IAM APIs.
>
>    You cannot call any STS APIs.

http://docs.aws.amazon.com/STS/latest/APIReference/API_GetSessionToken.html

>The temporary security credentials created by GetSessionToken can be used to make API calls to any AWS service with the following exceptions:
>
>    You cannot call any IAM APIs unless MFA authentication information is included in the request.
>
>    You cannot call any STS API except AssumeRole.

STS's AssumeRole (which we use with ``--sts-account-id``) works

http://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html

> The temporary security credentials created by AssumeRole can be used to make API calls to any AWS service with the following exception: you cannot call the STS service's GetFederationToken or GetSessionToken APIs.

**Comment by jantman** [comment]
done and merged to develop in #144
